pkg/monitoring/metrics/forklift-controller/metrics.go

```diff
@@ -0,0 +1,60 @@
+package forklift_controller
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+)
+
+const (
+	Succeeded = "Succeeded"
+	Failed    = "Failed"
+	Warm      = "Warm"
+	Cold      = "Cold"
+	Local     = "Local"
+	Remote    = "Remote"
+)
+
+var (
+	// 'status' - [ succeeded, failed ]
+	// 'provider' - [oVirt, vSphere, Openstack, OVA, openshift]
+	migrationGauge = promauto.NewGaugeVec(prometheus.GaugeOpts{
+		Name: "mtv_workload_migrations",
+		Help: "VM Migrations sorted by status and provider type",
+	},
+		[]string{"status", "provider"},
+	)
+)
+
+var (
+	// 'status' - [ succeeded, failed ]
+	// 'provider' - [oVirt, vSphere, Openstack, OVA, openshift]
+	planStatusGauge = promauto.NewGaugeVec(prometheus.GaugeOpts{
+		Name: "mtv_workload_plans",
+		Help: "VM migration Plans sorted by status and provider type",
+	},
+		[]string{
+			"status",
+			"provider",
+		},
+	)
+
+	// 'type' - [ cold, warm ]
+	planTypeGauge = promauto.NewGaugeVec(prometheus.GaugeOpts{
+		Name: "mtv_workload_plans_type",
+		Help: "VM migration Plans type",
+	},
+		[]string{
+			"type",
+		},
+	)
+
+	// 'destination' - [remote, local]
+	planDestinationGauge = promauto.NewGaugeVec(prometheus.GaugeOpts{
+		Name: "mtv_operator_destination",
```

Comment:
mtv_provider_destination? or migration destination (cluster)?

Reply:
isnt it the same? if the operator is install on a different cluster then the target it will be remote. but i"ll change to migration to make it more clear.

Reply:
the operator is the one responsible to have forklift resources in the cluster. on remote cluster you don't need a full MTV deployment AFAIK. the operator is irrelevant here in my opinion

Reply:
yeah, should be something like `mtv_target_provider`